web/auth/auth.go

```diff
@@ -529,6 +535,33 @@ func authorizeForm(c echo.Context) error {
 		return c.Redirect(http.StatusSeeOther, u)
 	}
 
+	// For a scope "login": such client is only used to transmit authentication
+	// for the manager. It does not require any authorization from the user, and
+	// generate a code without asking any permission.
+	if params.scope == oauth.ScopeLogin {
+		access, err := oauth.CreateAccessCode(params.instance, params.clientID, "" /* = scope */)
+		if err != nil {
+			return err
+		}
+
+		u, err := url.ParseRequestURI(params.redirectURI)
+		if err != nil {
+			return c.Render(http.StatusBadRequest, "error.html", echo.Map{
+				"Domain": instance.ContextualDomain(),
+				"Error":  "Error Invalid redirect_uri",
+			})
+		}
+
+		q := u.Query()
+		q.Set("access_code", access.Code)
+		q.Set("state", params.state)
+		q.Set("client_id", params.clientID)
```

Comment:
`client_id` should not be sent in the redirection. But I can see that I'm doing that also below.

Cf https://tools.ietf.org/html/rfc6749#page-26